packages/nuxt/src/module.ts

```diff
@@ -0,0 +1,25 @@
+import { addComponent, defineNuxtModule } from "@nuxt/kit";
+import * as onyx from "sit-onyx";
+
+export interface ModuleOptions {}
```

Comment:
There are `OnyxPluginOptions` that we could use here and also call the `createOnyx()` plugin.

But we can also do this in a follow up PR

Reply:
That's something to do once i18n and so on is integrated into this module. Like written in the description, this PR is just to bring the very basics of onyx into nuxt projects.